tracer/build/_build/Build.Steps.cs

```diff
@@ -448,13 +474,28 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
         .After(CompileNativeSrc, PublishManagedTracer)
         .Executes(() =>
         {
-            var (arch, extension) = GetUnixArchitectureAndExtension();
+            if (IsLinux)
```

Comment:
As before, split this into two targets IMO, PublishNativeTracerLinux and PublishNativeTracerOsx

Reply:
done in [bc27402](https://github.com/DataDog/dd-trace-dotnet/pull/3447/commits/bc274021c24a19f561fb286e35a68530bb086679)